task-sdk/src/airflow/sdk/execution_time/secrets_masker.py

```diff
@@ -130,6 +130,10 @@ def _secrets_masker() -> SecretsMasker:
     )
 
 
+def reset_secrets_masker() -> None:
```

Comment:
Can you add some docstring to say why a new processor type (if added in future) should use it and what it does

Reply:
Yeah will do